.prettierignore

```diff
@@ -7,3 +7,4 @@ dist/
 /.husky/_/
 .bob/
 .yarn
+supergraph-invalid.graphql
```

Comment:
Is this still required?

Reply:
Yes otherwise prettier complains because it cannot parse it as a GraphQL SDL :/

Reply:
GraphQL doesn't support prettier-ignore comment. At least, I couldn't make it work. I tried to add `# prettier-ignore on top.` but didn't work.